src/howto_private_tx.md

```diff
@@ -80,6 +113,18 @@ cd ~/work/substraTEE-worker/bin
 ./substratee-worker run
 ```
 
+Note: When launching the worker from within the docker environment, the following error is currently occurring:
```

Comment:
This error is not always happening and may only be related to our setup (driver and sgx sdk version). I would therefore put it under a Troubleshooting section.

Reply:
agreed